Python_Scripts/simulation_wrapper_amici.py

```diff
@@ -0,0 +1,169 @@
+import amici
+import numpy as np
+from time import time as toc
+import pandas as pd
+import os
+
+from loadModels import get_submodel_list, get_submodel
+from C import simconfig, DIR_MODELS
+
+def simulation_wrapper(
+        simulation_mode: str,
+        settings: dict,
+        model_name: str = None,
+        submodel_path: str = None,
+):
+    """This script wraps around the simulation with AMICI,
+    applies the settings for the ODE solver,
+    and returns a list of result, which depend on simulation_mode:
+
+    """
+    # get the list of sbml models which belong to this benchmark model
+    if submodel_path is None:
+        amici_model_list, sbml_model_list = get_submodel_list(model_name)
+    else:
+        amici_model, sbml_model = get_submodel(submodel_path)
+        amici_model_list = [amici_model]
+        sbml_model_list = [sbml_model]
+
+    # collect cpu times
+    average_cpu_times_intern = []
+    average_cpu_times_extern = []
+    # collect trajectories
+    trajectories = []
+    # collect failures
+    failures = []
+
+    # loop over models (=modules) to be simulated:
+    for i_model, model in enumerate(amici_model_list):
+        # get the adapted solver object
+        solver = _apply_solver_settings(model, settings)
+
+        if simulation_mode == simconfig.CPUTIME:
+            # we want to repeatedly simulate the model
+            n_repetitions = 40
```

Comment:
declare constant (potentially also in C.py, to allow lowering this number in the tests)